Good heavens, judging by the clock, we should be seeing a [Medbay Remap] dock.

FreeStylaLT opened the pull request:
Yes, it is that time again... again. This time an entirely new department. The one right in the center of the station. I think I might hate myself.

As the last time, here are the brass tacks for the impatient folk: (Click for increased size)
BEFORE:

![img](https://cdn.discordapp.com/attachments/195271472992878592/334353200780148736/medbaybefore.png)

AFTER:

![img](https://cdn.discordapp.com/attachments/195271472992878592/370571262193303552/updatedmapjesuschrist.png)

MAIN GOALS:

- Make treatment easily accessible and convenient
- Eliminate unused / unnecessary areas (Isolation + EV room thing (?))
- Move or otherwise change the layout so the non-essential (CMO, shrink, break room, etc) rooms are NOT in the goddamn center and / or way of everything.
- Improve the overall flow of the department.
- (Not actually a goal, but this happened as a consequence of shrinking the size of Medbay) add more maint
- Improve piping ( ***FUCK*** )

Let's get down to the DETAILS:

**New Lobby and Chemistry:**

![img](https://cdn.discordapp.com/attachments/195271472992878592/334675278540570624/chemandlobby.png)
I am convinced it looks a bit _worse_. I wasn't able to fix the aesthetic issue, unfortunately.
Anyway, it's been condensed into a single entrance, I know, three airlocks in a row, ew, but it doesn't seem to be that bad in reality AND it fixes the major issues with flow the department use to have.

The equipment of that cramped-as-fuck storage / locker room from current Medbay has been split into the reception, and other rooms, as you'll see.

As for Chemistry, it's been made larger.. greatly needed, from what I could tell.
They also received privacy shutters, while consequently getting more windows facing the lobby, to increase visibility, while protecting against the greytide, sort of, as it only blocks the lobby-side:
![img](http://i.imgur.com/B4VjDis.gif)

**Treatment / Main hall:**
![img](https://cdn.discordapp.com/attachments/195271472992878592/334671923206160384/treatmentcenterblock.png)

As mentioned, main point was to make treatment easily reachable and convenient, therefore pretty much everything related to healing has been stuck together into one giant complex.

Treatment's just _one_ room now with everything you might need, Cryo is right next to _treatment_ instead of Cloning - that's what it's meant to be used for.

Surgery also got attached and shrunken - having a ginormous 6x6 square for Surgery, and having _two_ of them seemed unnecessarily too much.

Each surgery also has its own storage room with no windows - both to reduce transparency and compensate for the loss of direct maint access (both surgeries still border maint with walls)

**Medical Storage, Morgue and Coroner:**
![img](https://cdn.discordapp.com/attachments/195271472992878592/334671937232175106/storagemorguecoroner.png)

Slightly expanded medical storage, added one more type of medkit into it (the useless, """advanced""" one, and also moved Syringe guns and Belts there - as part of the previously mentioned re-distribution. Having those rather high-demand items in the actual Medbay storage makes the most sense.

As for the morgue - heavily expanded morgue trays and a slightly more convenient access for the Coroner. Also, added direct access for Robotics. Convenience, mostly, but partly to keep disposal pipes from going through walls. Priorities, right.

**Cloning and Genetics:**
![img](https://cdn.discordapp.com/attachments/195271472992878592/334671934425923589/medbaycloningenetics.png)
Not much has changed, really, just slightly altered sizes and Cloning simply gets a cryo cell, instead of being attached to cryogenics, which both convenient AND cramped.
Cloning also gets a neat-o way of exiting:
![img](http://i.imgur.com/zbg76Il.gif)

**CMO and Break room:**
![img](https://cdn.discordapp.com/attachments/195271472992878592/334353194631299072/cmobreakroom.png)
As said, moved CMO away from dead-center of Medbay back to the sidelines, bordering Maint, while also giving the ability to overlook Genetics, Cloning and the Break Room, thus still giving some degree of importance to the room.
Still has the shutters:
![img](http://i.imgur.com/bblDKel.gif)

**Shrink, Virology, Maint, Secure Storage:**
In-Game:
![img](https://cdn.discordapp.com/attachments/195271472992878592/360799758379515915/unknown.png)
In-Map, plus Maint:
![img](https://cdn.discordapp.com/attachments/195271472992878592/360799957059633152/unknown.png)
Some new things, such as the addition of a sort of "Virology Lobby", which allows Virologists to be seen and talked to through the glass windows, however, the windows do have shutters. 

I believe this is a better integration of Virology into the core of Medbay, as it now appears to be an actual part of the complex, rather than an odd attachment, despite requiring quite the real estate.

![img](https://cdn.discordapp.com/attachments/195271472992878592/360799875471769601/2017-09-22_22-34-29.gif)

This is a bit of an experiment, with the goal being making Maint, or in this case, a part of it, easier to traverse for those who actually have legitimate reasons to be in it.

**Maint changes:**

Southwest Maint:
![img](https://cdn.discordapp.com/attachments/195271472992878592/360799917784170506/unknown.png)
A rather large change, incorporating some larger rooms and inter-connected smaller ones, another rather significant addition is the Engineering checkpoint:
![img](https://cdn.discordapp.com/attachments/195271472992878592/360799976395243520/unknown.png)
Its purpose is meant to serve as both a location for re-supplies and a shortcut in Maint, as an Engineer is able to skip going around the Bar entirely, while anyone without access (or tools) is forced to loop all the way around. 

![img](http://i.imgur.com/JuNZjXl.png)
Virology maint. Some of the new space has been re-purposed into an abandoned diner, coupled with a grill and a fryer, and some meat in the _personal locker_ in the corner.
As noticed by the odd-ass regular-floor tile in the south side and a welded locker up north, this utilizes the randomized assets for entries.

![img](http://i.imgur.com/xPNB3jz.png)
Attached secret maint rooms to xenobiology, both for the traitor potential and for the hidden room.. potential, which have been reported as somewhat lacking for our maint.

![img](http://i.imgur.com/mJVGu7D.png)
"reference" maint with some minor medbay loot (though a box of latex gloves isn't all -that- minor) which borders CMO and Genetics. However, the airlock leading to it is Medbay+ access.

Otherwise, I re-piped quite a bit around Medbay, though not enough to be really listed as a large change.

Oh, and, Medbay lost some misc crap like two (?!!) photocopiers and possibly a few roller beds.

That about wraps it up. 

🆑 FreeStylaLT
tweak: Medbay has been remapped
add: New Maint areas have been added south of Medbay.
/🆑

alexkar598 commented:
too cramped surgery is too small and me is seeing no medical fridge

DesolateG commented:
@alexkar598  look at the medkit storage room.. it's there.

Da-Dman234 commented:
Why the decision to remove the shutters from surgery? Pretty big hit to antag MD's if you ask me, especially since you decreased the size of medbay overall

SamHPurp commented:
I will methodically put together feedback for you soon as I can see some issues...

But one immediate thought is - Everything important is too compacted. One single syndie macrobomb will remove all of medbay.

FreeStylaLT commented:
@Da-Dman234 Apologies! I missed the Surgery shutters. They're there now.

FreeStylaLT commented:
@SamHPurp That is a consequence that comes by design in return for convenient and easier accessibility of the important areas.

SamHPurp commented:
@FreeStylaLT and thats fine, but that is going to mean shuttles called more frequently due to a lack of a functioning medbay. So as long as that trade off is well understood.

alexkar598 commented:
Could you move the med fridge between chemistry and medalist storage,

alexkar598 commented:
Medkit* damn mobile keyboard

SamHPurp commented:
Can you see if this works?

https://redpen.io/dh8cc2e0158b3d193a

FreeStylaLT commented:
PR updated with various minor adjustments:
- Removed curtains from treatment
- Adjusted lighting in morgue and in other areas
- Adjusted hallway floor patterns
- Added missing (oops) Surgery shutters
- Revised a bit of Maint, reduced their open areas
- Reduced Medkit storage to one door for entrance

All relevant pics should be updated

TravellingMerchant commented:
Here's a question, can we still get those cute little union jack mugs here? 

Drinking tea out of the bar's glasses feels wrong.

pinatacolada commented:
the important bits feel way too cramped overall:

I like the lobby a lot, just maybe remove the floor markings for the lines, they feel a bit unneeded and like visual clutter

There's way too much space in the corridor next to the emergency entrance, it's way too wide, feels empty, feels bad. The two beds with the IV and box in the middle seem like they got dropped there out of nowhere, feels out of place and messy

The chemistry, storage, and morgue are a god sent and I wouldnt change a thing.

Genetics looks ok to me, the monkey pen is maybe a little too small, but who cares nobody likes geneticists anyway

I do take issue that there is no isolation rooms. Yes, they are underused in the current map, but we still need a place to ~~lock up prisoners~~ keep infected separated from everyone else and in a place where they can't wander off while the cure is made for their GBS or wtv. That said, there isn't much room to play around with these so its wtv to me

Now for the really big elephant in the room

I like the tiny prep room thing for both ORs, but I think they could be just slightly wider, maybe just a single tile or two at most, I will expand on this later and justify why

This whole block is a mess and I honestly hate it with a passion, here's why:
![capturar](https://user-images.githubusercontent.com/13833056/28083399-1a7fcfac-666d-11e7-94e8-15d649179a6f.PNG)

It's ultra cramped

Flow wise it makes absolutely no sense, here's two examples:
-If you are carrying a patient to surgery, you need to go through other people that are being treated in the sleepers

-If you just want to watch your friend go through liver surgery, you need to go through people being treated in the sleepers

It's a bit hard to put into words what I feel about it but here's what I see is wrong. On the top you have one room for one thing. To access the surgery area, you are forced to cross this room. In my head this shouldn't happen, there should be corridors to access rooms, not going across a room to reach another:
![capturar](https://user-images.githubusercontent.com/13833056/28083770-726e9a12-666e-11e7-8763-4ab1a729158c.PNG)

I really really dislike that tiny freaking room with the sleepers, and I want to nuke it from orbit tbh

What I would do to fix that whole mess is something like this:

![capturar](https://user-images.githubusercontent.com/13833056/28086162-fb86022a-6675-11e7-817e-571e030d2d23.PNG)

We would now have a place to handle patients that take a little waiting time (ie drunktards that you have to wait on pentetic to nuke their vodka, people with the flu so you can keep them in a tiny room while you fetch a cure, etc)

You would no longer need to be crossing rooms to get to another room

paramedic regains a door to medbay

removes the unnecessary white space after the emergency entrance

adds a really fast route from emergency entrance to surgery (internal bleeding cases and so on), which increases the value of having the emergency entrance, in theory

I'm not totally satisfied with the corridor from the entrance to the surgery prep room though, feels a bit empty, could maybe do with a potted plant, a wall medvendor, or some fluff like that

Landerlow commented:
I like the changes, I personally think the OR's are fine. The only real issue I have is that the medical fridge is still easily accessible for non-medical personnel. This bugs me to no end. Non-medical personnel should not have easy access to medication. I know that a lot of the server population has gotten their medical degree from the internet and therefor know, for example, that mannitol heals their brain, when they are a clown by profession. Please, I would *love* a medbay where the fridge is actually in a secure area, only accessible by medical personnel.

Fox-McCloud commented:
Absolutely no BODA machine on station.

It's a derelict Russian-station only object.

- Why does medbay have a welder?
- Why is there a spare set of surgery tools?
- There really shouldn't be mining gear in maintenance
- There really shouldn't be a spare botany sprayer in maintenance either; I will loot this every single shift just to cause people to slip at my whims, just 'cause I can
- Sustenance vendor is a gulag and permabrig only thing
- Do we really want a public kitchen? 
- Floating fire extinguisher in Virology
- Virology is probably the most secure place on the entire station now (not sure if good or bad, but worth pointing out)
- Donk pockets everywhere is probably a bad idea
- Loading the fridge is going to be more of a pain in the ass than it currently is
 - I'm tempted to say the fridge should just be accessible to everyone in medbay lobby, or at least there should be two fridges, mayhaps
- The two orange medkits are unnecessary in medical storage; each doctors spawns with one of these and there's an over-abundance of them
- the curtains on surgery tables are going to be annoying as hell
- cloning needs a button so you can leave; similar to medbay itself.
- what in the world is next to the creepy red Bay doll?
- RIP Geneticist antags ever having any privacy, ever
- Is there a box of medical glasses somewhere still?
- Excluding medical doctors and detectives from performing autopsies is bad
- I hope that ID card in maint isn't an agent ID card

Vivalas commented:
I dislike the removal of photocopiers from medbay, especially from the CMO and coroner, the two jobs where a copy machine is most relevant. All head offices should have one for bureaucracy and coroner is really only fun because paperwork.

Also at least one general access copier in medbay would be nice for doctors who enjoy a little paperwork or RP. Removal of photocopiers is a big no.

Rurik123 commented:
I really like the overall look, barring some little things a couple others mentioned.

SamHPurp commented:
Booted this up to do a deeper review... and it runtimes:

```
[22:34:33] Runtime in random.dm,25: Cannot create objects of type null.
  proc name: spawn item (/obj/random/proc/spawn_item)
  src: Random Therapy Doll (/obj/random/therapy)
  src.loc: the plating (139,96,1) (/turf/simulated/floor/plating)
  call stack:
  Random Therapy Doll (/obj/random/therapy): spawn item()
  Random Therapy Doll (/obj/random/therapy): New(the plating (139,96,1) (/turf/simulated/floor/plating))```

EvadableMoxie commented:
I don't really think the current Medbay needs a massive revamp.  The only problems with it is cloning/cryo getting cramped when things get busy, and maybe the morgue being a bit too small.  Beyond that the current Medbay works so I don't see any reason to go changing it. 

The lobby looks really strange aesthetically, and it doesn't even have chairs which strikes me as incredibly odd.  This isn't really a huge deal, though. 

I really don't like the surgery area. The current boxstation surgery rooms are perfect.

The current surgery is at the end of an open hallway and you get in via a 3 tile wide door.  As you enter, you have a view of both ORs so you know if one or both is occupied.  If one is, it's easy to go to the other, if both are, you are already where you need to be to wait.  The ORs are just the right size, not at all too big from my experience. And the waiting area makes it easy to get out of the way and doubles as a storage room.

In this design,  you need to go past a cramped room with 2 sleepers and then pass a 2 tile long 1 tile wide choke to get into the prep room. Since we can't move diagonally that means there are 4 tiles that, if blocked, bar passage.  This is awful. Surgery needs to be very open because often surgery needs to be accessed quickly, and you often need to bring people in who can't walk without assistance or might need to be rolled in on beds.  It really needs a 3 tile wide entry, in my opinion, but even if you disagree with that, a 2x1 choke is just way beyond acceptable.

Instead of making both ORs slightly bigger and storing the things you need where you need them, the ORs are smaller and storage is in entirely separate rooms. So instead of making surgery 4-5 tiles bigger, there are now two new 8 tile storage areas that make getting what you need less convenient, in addition to 2 extra rooms that will barely ever be used.

Chem being windows instead of walls is a downgrade to it's security, as is having a door directly into chem instead of having to get past the storage room.  Yes, the chemist can use the shutters if someone tries to break in while they are there and paying attention.  The problem is if the chemist is away, getting in is now much easier.

The chem fridge is way too far away from where the chems are made.  To the left of the door leading into chem would be a better place for it. 

Medbay lost a scanner since the exam room is removed, which isn't a problem for Medbay itself, but does hurt the Brig Physican who often takes that scanner for BrigBay.  That is something that should be considered, I think.

All in all, I'd personally rather see some tweaks to the current Medbay that help the general Exam Room/Morgue/Cryo/Cloning areas rather than rebuild the whole thing when for the most part, it works.

SamHPurp commented:
I like that sometimes using my maintenance spawners I added! :)

Slice of maintenance has the wrong `area`:

![image](https://user-images.githubusercontent.com/7415122/28092108-294dd57a-668a-11e7-824d-5be0e8e42d08.png)

OR1's backroom has the walls as part of the exam room for some reason:

![image](https://user-images.githubusercontent.com/7415122/28092141-478fa572-668a-11e7-88ac-15df6670df51.png)

One of the CMO's office tiles is Genetics and I couldn't see why: 

![image](https://user-images.githubusercontent.com/7415122/28092182-67c2731a-668a-11e7-8d87-e25aba79f0b4.png)

Lightswitch in Coroners office could easily be by the door, rather than 4 tiles deep.

All of these are set to 4 pieces of maint loot... thats a LOT of maint loot to spawn there:

![image](https://user-images.githubusercontent.com/7415122/28092382-1071a756-668b-11e7-9507-b5bc3bf6969d.png)

Viro extinguisher was mistakenly placed wrong:

![image](https://user-images.githubusercontent.com/7415122/28092412-2e9f840a-668b-11e7-8d35-69c906ef1e56.png)

Too many lights together on viro hallway... some should be removed, or all made bulbs or something:

![image](https://user-images.githubusercontent.com/7415122/28092499-7fffc300-668b-11e7-9e21-17310686284a.png)

Bulbs in morgue look way better, it was a good change, but it could use some more on the north/south walls I think. Looks TOO dark ingame.

You have 2x boxes of papercups on the medbay reception desk. Since the water cooler now dispenses cups these aren't needed. Or atleast certainly not TWO boxes of them.

The OR lightswitches do very little, because you've got a bar light on the window in the observation area. Lights on glass don't look good anyway, and you can't place them like that in the game... so it would be good to move these.

Missing quarantine blast doors in Chem's windows. Its in the windoor, but not the window like Receptions is.

Could do with another spray bottle, perhaps down by the bacta tanks or the exam 'room'

IcyV commented:
Most of my concerns have already been mentioned but a point I wanted to bring up is the location of the lower chem setup in chemistry. I'm not too big on the idea of a chemist basically being able to hide that far down from the front desk if they don't feel like talking to people. A lot of chemists already pretend nobody is there so they can keep working on their personal project. 

I think glass windows and shutters would only help to spread this problem further as they could justify a broken window as closing the shutters for as long as they want. It opens up chemistry to being purely department focused instead of helping the public.

Cid777 commented:
Chemistry really does need shutters, and it's worth the "reduction" in security by replacing the walls with windows. It's in a fairly public area, you're going to be noticed if you break in.

I like Robotics having direct access to the Morgue, although it looks weird being so massive.

@pinatacolada There are still isolation rooms inside Virology, although you probably means ones accessible by MDs.

Cid777 commented:
> It opens up chemistry to being purely department focused instead of helping the public.

Is this really a problem? Chemistry should be focusing on medicine, that's their primary job. Asking Science for chemicals isn't unthinkable.

Xantholne commented:
My current concerns for this are:

- The ORs feel too pushed up into the rest of medbay and makes the immediate area look cramped especially when you have massive amounts of dead, docotrs with oher patients etc all pushing laong trying to use the immediate sleepers/cryotubes. A better solution would be keeping the sleepers and cryotube rooms still more seperated. and moving the OR rooms down a few tiles.

- The Paramedic closet is also in a weird area now, it has nothing actually connecting it to the rest of medbay nor a side entrance into medbay which they sometimes used to actually get in and out with patients

- The Medfridge is still an issue being in a publicly accessed area. I guess *A* solution would be adding medical access requirements to the board itself, but honestly sometimes you have to self serve yourself in medbay especially when it's understaffed/overwhelmed

ktoma36 commented:
Just a note from an atmos wizard. I personally love the piping you have set up:

- Having both the distro and scrubber pipes running alongside each other helps IMMENSELY when trying to fix the dam place when stuff inevitably explodes.

- Virology is now connected to the medbay's pipe system rather than oddly attached to the maint loop heading to science. 

- With viro attached to the medbay system the emergency oxygen supply actually supplies medbay (if any of you were wondering what that little atmos only room bellow virology is and the random atmos rooms around the are, those are emergency tanks of oxygen in case atmos ceases to function for some reason). 

- Virology has access to the space disposal loop. 

- More air alarms spread throughout medbay. This is really useful considering the fact that with the old set up only one air alarm (the one between medbay storage and the CMO's office) for nearly all the firelocks around the CMO's offce to drop closed. 

- No disposal pipes are going through walls (except for virology's, but that is never a problem. 

- Removed the windows to space from the halls and operating rooms. No more easy way to depressurize medbay!

A couple of things that would be nice to add:

- Piping and electrical stuffs to the asylum maint area. Seeing as it used to be part of the medbay proper it would be odd for it not to have these systems in place. Perhaps having the wiring and pipes incomplete, as well as having the APC and air alarms damaged/ not fully built. If any light fixtures were to be added non should have working bulbs.

- The paramedic's office vent could have its distro pipe ran up along the scrubber's pipe through the maint door

FreeStylaLT commented:
@Fox-McCloud 

   >Absolutely no BODA machine on station.

Removed. 😢 

    >Why does medbay have a welder?

It doesn't. That's the glove-snipper. I use FastDMM to take the large screenshots, which seems to use a welder sprite for it.

    >Why is there a spare set of surgery tools?

Including a back-up set of surgery tools has been suggested by some players, due to constant losses to both docs and thieves in general. It's a change I'm skeptical of myself, due to the removal of possible inter-department interaction. 
However, I also considered the new location and size as making theft in general easier - both due to surgery being attached to a secluded Med-only maint and due to it being more easily accessible for everyone, therefore, I decided to keep it to alleviate those possible downsides.

Not really a big deal to remove, though, if that's what's decided on.

    >There really shouldn't be mining gear in maintenance

There isn't. It's Fast DMM changing the bear trap's sprite (which it doesn't have in DM) into... that.. mining thing.

    >There really shouldn't be a spare botany sprayer in maintenance either; I will loot this every single shift just to cause people to slip at my whims, just 'cause I can

Removed.

   >Sustenance vendor is a gulag and permabrig only thing

Removed. :cry:

    >Do we really want a public kitchen?

Maybe. A 'diner' type of abandoned room was made as a suggestion to put in the available maint space. It does somewhat contribute to the mentality of "never having to leave maint", as now you can both get drunk AND eat off of maint, which.. may not be a desirable precedent / outcome. Maint was not a priority for me, so I'm fully open to any and all suggestions regarding Maint.

    >Floating fire extinguisher in Virology

Fixed.

    >Virology is probably the most secure place on the entire station now (not sure if good or bad, but worth pointing out)

Probably. Though, it does have the same atmos-closet weakness, and it is still accessible through maint, you just might need a few more tools and time for it. I don't see much benefit, or loss, in having Virology be secure, considering its lack of use one way or the other, though, this may affect xenos.

    >Donk pockets everywhere is probably a bad idea

Agreed! Removed them from both the store and the diner. Yes. Find them elsewhere, come to the diner to cook.

    >Loading the fridge is going to be more of a pain in the ass than it currently is
        I'm tempted to say the fridge should just be accessible to everyone in medbay lobby, or at least there should be two fridges, mayhaps

I've seen Stations have that, actually, one fridge available to the public, in the lobby, and one in Medbay proper, in a secure area. I'd be willing to experiment with that.

    >The two orange medkits are unnecessary in medical storage; each doctors spawns with one of these and there's an over-abundance of them


    >the curtains on surgery tables are going to be annoying as hell

I just added them for immulsions. :cry: Removed.

    >cloning needs a button so you can leave; similar to medbay itself.

Added.

    >what in the world is next to the creepy red Bay doll?

A fan. We have that. It blows and everything.

    >RIP Geneticist antags ever having any privacy, ever

After moving CMO out of dead-center, where it kind of overlooked everything and nothing at the same time, having it overlook job-rooms made the most sense. Cloning and Genetics seemed to be one of the more important / expensive areas to oversee. The two windows going from CMO to Genetics can be removed, though. Not a big deal.

    >Is there a box of medical glasses somewhere still?

Added, though, they aren't present in current medbay, either, oddly.

    >Excluding medical doctors and detectives from performing autopsies is bad

I do not think that is the case for MDs, as Coroner's access is just basic Medbay access.
As for the Detective, I've changed the inner Coroner's door to be accessible by either Detective, or regular MD access.

    >I hope that ID card in maint isn't an agent ID card

It's a blank ID card. Really kind of useless as an item unless you already have HoP access.

@SamHPurp 

**Changes:**

Fixes:

- Fixed Paramedic maint areas
- Fixed OR1's areas
- Fixed Virology's extinguisher
- Fixed CMO's area
- Fixed minor issues in Viro maint
- Fixed Chemistry not having Quarantine shutters
- Fixed Experimentor disposals

Changes / Removals:

- Removed the red doll from Medbay-only Maint
- Removed the bear trap from the store
- Reduced the store's loot spawns to be 2-3 instead of 4
- Changed BODA and Sustenance Vendor to Cola and Hot Drinks
- Moved the medfridge to inside the storage
- Removed the pill cabinet, instead, moved all of that crap in it onto the southern table in the treatment room.
- Removed the western table in treatment room. Now both sleepers have 3 tiles around them that do not interfere with the center path leading to Surgery. This should help with some of the flow issues, meanwhile, I'll see about an alternative, or increasing the size of treatment.
- Amount of lights in the secondary medbay hallway have been reduced
- Diner no longer has syndicigs (yes, that was a syndicig packet)
- Remapped the treatment room to be more spacious, made Surgery obs. room have double airlocks, expanded it by one tile
- Removed the lounge as to make room for above changes.

Additions:

- Added a button to open Cloning
- Added a second medifridge to the lobby. Let's see how that works out.

As for @pinatacolada and others who have expressed cramped / flow issues:
I disagree, partly.

Currently there is a very easy central path going through treatment to Surgery, which, to my belief, should not interfere with the sleepers, and only possibly, that is, a single tile out of the 4 adjacent to the scanner, out of all of them, interfere with the scanner.

Regardless, though, I did make some changes in the hopes of alleviating some of the issues you listed.

Also, your proposed solution is a bit cumbersome to utilize and goes against my goals of an open, inner-public treatment area, and doubles the amount of sleepers available, while providing its own flow issues:

![img](http://i.imgur.com/CR1c5G6.png)
It has poor flow from more-or-less every entrance that is currently there, has two sleeper-rooms that are completely out of the way and isolated, has a superfluous additional hallway on the left and none of the sleeper rooms has easy access to cryo.

In order to make something like this feasible, and look good, I would more-or-less have to remap the entire Medbay again, altering the lobby to fit the new flow, consequently altering the reception and chemistry, which consequently would alter morgue, coroner and medkit storage, which would consequently alter cloning and genetics, which would consequently alter the CMO's office, and so on.

I do get what you want to go for, but that is not a direction I would like to see, unfortunately. Convenient flow of lobby / side-entrance to treatment area to cryo / surgery is what I seek through open rooms, rather than your suggestion.

ktoma36 commented:
Im still drooling over the pipes :3. Also, we have fans?

FreeStylaLT commented:
OOPS. Forgot to remove the dev pastas. Quick commit coming up.

SamHPurp commented:
@FreeStylaLT > dev pastas

Random bits of the station in space? Glad I'm not the only one...

deep-fried-faceplant commented:
Currently, people breaking into medbay through the lobby by smashing windows have to go through medical storage, where high priority items like medical belts, medical HUDs, and syringe guns. New medbay can be smashed into through the lobby window or the west entrance now, which initially seems like a buff for greytiders, but at least prevents them from grabbing important medical gear on their way into medbay.

Most medical treatment done is either at the sleepers, cryotubes, or the few tiles surrounding the medical fridge. That table in the sleeper treatment room should probably be moved out of the way to make room, perhaps moved to where that potted plant is.

Crates with space to move around them will get kicked around. The bloodbag crate next to the IV drips and beds should be flanked by windows, or just moved to medical storage.

I really like the changes to maintenance, especially the removal of that long 1 tile wide tunnel west of virology. 1 tile wide hallways make melee combat pretty much impossible, give 1-hit weapons a stupid amount of power, and make maintenance feel far too cramped than what it should be. 1 tile wide hallways should be reserved for alternative routes or if there's no better way to go about creating a hallway.

Ty-Omaha commented:
@deep-fried-faceplant 
> Currently, people breaking into medbay through the lobby by smashing windows have to go through medical storage

They can break the reception windows and use the door control buttons on the desk

Vivalas commented:
@FreeStylaLT Please add photocopiers to CMO, coroner, and possibly even one somewhere in the medbay, photocopiers shouldn't be removed.

SamHPurp commented:
Overall, I feel as though the reduction in size and the compression of vital areas is counter-productive, and enables medbay to be disabled far too easily with single canisters of plasma, bombs... or even just minor breaches on fastmos.

I'm glad people like the Coroner, but this gives the Coroner the most space of ANY medical role by far... and space I don't personally feel he needs.

It feels like the plan was more to expand maintenance, which has been accomplished, and looks like it'll add some interesting gameplay... but to do it you've had to eat into Medbay. I do not mean any offense there, the maintenance bits look good, but I'm not sure I can buy the reduction in Medbay size.

ktoma36 commented:
I do like that surgery isnt open to the full maint in this build. Nine times out of ten those doors are just used by antags to capture knocked out surgery patients for a quick snack/murder/mindslave/facehug.

Vivalas commented:
@SamHPurp Coroner is currently quite cramped, that's one of the changes here I actually kinda like. Still, the removal of the photocopier from the coroner's officer screws the coroner over a lot more than the extra space helps.

FreeStylaLT commented:
Any maint-related changes were a complete byproduct of the design, I had zero intentions to expand Maint, with one of my earlier iterations literally eating into it so much that the Construction room in Engineering had to be shrunk / almost removed.

EvadableMoxie commented:
Surgery is better but you still have to go past the sleepers to get there which will be a bit cramped.  I also greatly dislike the separate storage rooms for the OR.  They waste space and time by arbitrarily placing things you need in surgery in a separate room, whiling making the ORs less secure.  If you just get rid of them, the space taking up by the storage room above the left OR would be available.  The sleepers could be moved into that space, clearing up the entrance to surgery. The rooms could just be made bigger to accommodate the extra items, as they are now.

Ar3nn commented:
>making maint "shortcuts" and generally doing anything to make it feel not-cramped

👎 x 💯

alexkar598 commented:
Remove the ton of medhud on the table doctor lockers each have a med hud

FreeStylaLT commented:
Fixed conflicts, otherwise, didn't make any layout or other major changes, beyond some minor pipe optimization.

gingeralesy commented:
What was the reasoning to have full surgery gear in a locker in the maintenance? The small room with the safe that seems separate from medbay. It seems a bit odd. Would fit better on the larger room above it that's connected to medbay.

Speaking of connecting to medbay, I noticed there are no doorways into the medbay from the maintenance. A bit odd, considering the frequency of "Radiation Belt" events. Unless that room below surgery is shielded. Still a bit far away from the northern bits of the medbay since that tiny maintenance between morgue and robotics is gone.

I love the abandoned restaurant, by the way. So renovating that.

I also appreciate the decision that CMO's office has only regular walls between it and the maint.

ParemberKennard commented:
I really like the new ORs. There's a lot of wasted space in the old version, this puts everything in arms reach, meaning you don't need to do that awkward side shuffle shit in the middle of surgery. The public med fridge is also very convenient crew-side asset that I missed from other servers.

That being said there's one major issue I found. I'm pretty sure the paramedic's garage being so broken now after implementation is a side effect and not an oversight, but it still needs noticing. There's a weird sliver of maint tunnel seperating it from medbay proper, and the garage door buttons are on the airlock leading inside.

Jovaniph commented:
Great job on the piping work. I'm still left triggered by some of the piping going through walls and not the doors.

Jovaniph commented:
South Medbay maintenance is now a cluster fuck...why not just make an opening to space?

tinfoiltophat commented:
That chemfridge is in a bit of a weird place. You have to like wrap around to get to it. I think it'd be better to the left of the fire alarm outside chem, where that window is.

Balthusdire commented:
I love it. Some people say it is missing some minor things and I dont know what those are, but overall I adore the new layout.

EvadableMoxie commented:
As a psychiatrist, I had access to the paramedic garage from maint but not from Medbay, I assume that's an error.  Also, I had access to the Coroner's office, but not the morgue.

I'd recommend getting rid of the disposal unit at the corner where the hall turns down into the CMO's office, psych office and virology.  It blocks the corner so that it becomes a 1 tile choke, and it's redundant.  There's a disposal at the cryotubes, in the break room, and in genetics, we really don't need one there.

Going to need some sinks somewhere, too.

AffectedArc07 commented:
PLEASE FIX THAT ONE BRUTE TRAUMA KIT WITH A DIFFERENT TEXTURE. IT HAS PISSED ME OFF FOR TOO LONG NOW

TDSSS commented:
As someone that plays Roboticist a lot, I really dislike how this basically robs Robotics of its maint area. I mean, that tiny little maint corridor was already very pathetic, but removing it just makes it worse.

FreeStylaLT commented:
Some updates after a totally brief hiatus:

- Moved Virology and re-remapped it, it has shutters and glass now.
- Tweaked Chemistry
- Tweaked Morgue and Coroner
- Tweaked medical storage
- Moved chemfridges, added glassdoors for them
- Re-did Maint

Some pics were updated, mainly the **Virology / Maint** ones and the **whole-map one.**

Will get around to updating the individual room images tomorrow.

Jovaniph commented:
Wow. I like the Virology Area. I still don't like maintenance all the much. The abandon bar just seems out of place. Honestly, I just keep seeing more and more potential for Atmospherics to take all that space. If I don't procrastinate, i'm thinking about doing a map edit after this one. (if this gets merged)

taukausanake commented:
@Jovaniph Why does Atmos need any more space than it already has? I wouldn't expand Atmos into the maint halls. That bar is a little out of place though. Maybe turn it into a different kind of room like abandoned Hydro or library with some minor loot. Better than a second abandoned bar

Jovaniph commented:
@taukausanake Ohh. I don't mean expanding atmos. Just readjusting or moving it.

FalseIncarnate commented:
Its an abandoned diner, not a bar. It has cooking machines, not booze.

Also @FreeStylaLT you mad man, I see what you put there (or you used some old pictures)

PacifistDalek commented:
Loving it overall, especially the new genetics setup, it's gorgeous. The red phone in the lobby left room should probably move as it blends into the stack of stethoscopes on top of it. (just for aesthetic reasons)

tylerh95 commented:
Love it! Has some minor bugs, but it's a much needed fresh face for the Medbay.

uraniummeltdown commented:
The new Virology is really bad, Virology and Xenobiology are the only places on the station where you can do shady stuff easier due to their isolation. Viro especially is one of the safe havens for ventcrawling antags like xenos/swarmers/borers. Virology should not have windows at all, replace those with reinforced walls. Hell it doesn't even need a lobby.

Jovaniph commented:
I feel like if we are going to make every decision on how a map should look like based on antag advantages and disadvantages. There will be no reason for new map or map changes. Can we allow people to adapt and conquer or is there have to be a set of rules that need to be follow when making/changing maps?

DesolateG commented:
@Jovaniph antags need to be taken into consideration when designing a map. There needs to be spots on it to be able to do things like this. Such is why old bar, electrical maintenance, incinerator, turbine, old surgery, etc.. exist.

Also, the idea behind virology is supposed to be that it's a very quarantined area that's hard to breach. After all there are dangerous viruses being grown in that lab.

Doukhan commented:
TBH virology ATM is such a good place to do shady shit that it's a bad place to do shady shit. People know to check that place often, the turbine/electrical storage have the same issue. If anything making it less of an obviously superior p;ace to get up to antagness might actually make it better if you get what I mean.

FreeStylaLT commented:
- Changed the maint ***"bar"*** to look at least somewhat more like an actual diner / restaurant
I'm starting to think I should really replace the diner with something else, but I don't have that many ideas for larger rooms that would go in the southern part of maint.
- Various minor tweaks and fixes

Quantum-M commented:
The psychiatrist is rather more isolated than before, I think it should be closer to the Medbay entrance (for those quack patients)

Sikroth commented:
![missing_pipe](https://user-images.githubusercontent.com/7857084/31583138-e726d1e6-b194-11e7-849c-357c119974cc.png)
There's a missing piece of distro pipe right under the maint door under the Janitorial closet. Unless someone removed it during that round for some reason..

bigfatbananacyclops commented:
In my opinion i think we should have a more decent blood storage room.

Jovaniph commented:
This Light switch is badly placed https://gyazo.com/a9eb631e55733cd7bf0da9217809dc8f

Jountax commented:
If you really want to replace the diner, here's a few things that you could put there:

Old Fishery/Fish Farm, complete with unpowered critter care and some tanks/wall tanks.
Old Xenoarch Lab, because that job no longer exists.
Old Dressingroom for the clown/mime, full of props and costumes.
Old barbershop, with another dye machine and some mirrors, good in case someone steals the barber's machine for some reason.
An old arcade, with a claw machine and some other machines. Maybe powered, maybe not.

IntentEldiablo commented:
One thing that will need updated is the camera network, looking at medical on security monitors its the old layout and the cameras are still in the old positions, lots of blank spots but thats about it

Blazitron commented:
I think chemistry could get some working tables rearrangement, so chemists are able to turn around in their chair and place beakers and that sort of thing, without having to move from place to place, because right now they are too far away from the workings stations...

Dumbdumn5 commented:
A maint entrance on the east side of medical might be good for the sake of consistency, as every single other department has a maint entrance on both the eastern and western sides. Currently, to get from med maint to experimentor maint the closest exist is next to the shrink's office, meaning that the player would have to walk all the way around the disheveled engineering, shop, and bar areas south of the psych's office. I'd suggest a maint entrance just to the right of where the washing machine is in the break room now, moving the computers to be aligned with the vendors, and giving room for the washing machine and air alarm to move down a tile. The stool can be placed over the room vent or removed totally if it looks too cluttered.

Alternatively, you could move the red phone to the metal tables, remove the wooden table, scoot the washing machine up a tile and move the air alarm to the opposite side of the wall that the CMO request console is, then place the maint entrance in the same area just behind where the washing machine would've been.

I ask this mostly because the current route from west med maint to east med maint is incredibly long compared to the previous maint rendition, and it would benefit travel greatly by opening up more routes than a good 80 tile stretch of maint halls till reaching the nearest eastern departmental airlock. That or it could be good to add an airlock to the wall by the medistan flag spawn, as it'd cut travel distance by at least a little bit.

StreakyHaddock commented:
Cryogenics needs a screwdriver, wrench and cell charger for the defibrillators.

Da-Dman234 commented:
Paramedic's office doesn't have any lights and the garage door button isn't accessible when in the office

Jovaniph commented:
This is also a thing https://i.gyazo.com/e5dadc54db037cccb3a894d4d4901c09.mp4

This include the room below which is not in the gif.

taukausanake commented:
@Jovaniph Wouldn't that be fixed by making the wall not part of Xenobio area? That way it wouldn't be able to see into the room next to it? That's the very limit to my understanding of areas though

Jovaniph commented:
@taukausanake you are absolutely right. It still requires a .dmm change however.

FreeStylaLT commented:
Lots of updates and fixes, main (huge-ass) image updated, the rest will come later.

taukausanake commented:
I think I can get behind this map change now. Lots of work being put into it, I think it'll be fine if not better than what we have

Doukhan commented:
The only little thing that bothers me is that there's only one stool in the lobby, not much of a waiting area with only one place to put your butt.

Jovaniph commented:
You cannot access the Chem Heaters... Also, the chem dispensers can't be upgraded (Not sure if it is intentional. I would rather scientist build a new one). Lastly, no screwdrivers.

I am actually clicking
https://i.gyazo.com/4c131510dd25a6abe2dca593c648dee2.mp4
https://i.gyazo.com/70f8df9e5939739de19efc87f110ee98.mp4
